dockerHashing/src/main/scala/cromwell/docker/registryv2/DockerRegistryV2Abstract.scala

```diff
@@ -121,8 +127,12 @@ abstract class DockerRegistryV2Abstract(override val config: DockerRegistryConfi
     * @return docker info response
     */
   protected def getDockerResponse(token: Option[String], dockerInfoContext: DockerInfoContext)(implicit client: Client[IO]): IO[DockerInfoSuccessResponse] = {
-    val request = manifestRequest(token, dockerInfoContext.dockerImageID)
-    executeRequest(request, handleManifestResponse(dockerInfoContext, token))
+    val requestV2 = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptDockerManifestV2Header)
+    def requestV1 = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptOCIIndexV1Header)
```

Comment:
would require more variable renames below

```suggestion
    val requestDockerV2 = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptDockerManifestV2Header)
    def requestOCIV1 = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptOCIIndexV1Header)
```